pkg/controller/scylladbcluster/resource.go

```diff
@@ -434,6 +434,44 @@ func calculateEndpointsForRemoteDCPods(sc *scyllav1alpha1.ScyllaDBCluster, nodeB
 			})
 		}
 
+	case scyllav1alpha1.BroadcastAddressTypeServiceClusterIP:
```

Comment:
Since we didn't notice it's missing in the first iteration lets add unit test coverage to this function.